game/action.go

```diff
@@ -5,13 +5,15 @@ import (
 )
 
 type Action struct {
-	Type          ActionType
+	Type ActionType
+
 	Card          *Card
 	Cost          *Cost
-	Source        *Permanent // for targeted effects
-	With          *Permanent // for attacking
-	Target        *Permanent
+	Owner         *Player
```

Comment:
What is the owner of an action? Is it the player who is taking the action or the player who owns some card that is acting or what?

Reply:
(If it is the player taking the action how about calling it just Player, to make it less unclear when there is a controller vs owner distinction)